assets/queries/openAPI/success_response_code_defined_patch_operation/query.rego

```diff
@@ -0,0 +1,24 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+	response := doc.paths[n][oper].responses
+
+	oper == "patch"
+
+	object.get(response, "200", "undefined") == "undefined"
```

Comment:
This does not works as the same, because we want to get a result if no success code is declared, with the array technique, each undeclared success response will generate a result...